app/services/kredits.js

```diff
@@ -149,45 +144,47 @@ export default Service.extend({
   getContributors() {
     return this.get('contributorsContract')
       .then((contract) => contract.contributorsCount())
-      .then(contributorsCount => {
-        debug('[kredits] contributorsCount:', contributorsCount.toNumber());
+      .then((count) => {
+        count = count.toNumber();
+        debug('[kredits] contributors count:', count);
         let contributors = [];
 
-        for(var id = 1; id <= contributorsCount.toNumber(); id++) {
+        for(var id = 1; id <= count; id++) {
           contributors.push(this.getContributorById(id));
         }
 
         return RSVP.all(contributors);
       });
   },
 
+  replaceIpfsHash(data) {
```

Comment:
was wondering about the naming here... as it is more like building the ipfs hash - but then it is also replacing the "ipfsHash" attribute.